nd4j/pom.xml

```diff
@@ -417,150 +402,6 @@
         <profile>
             <id>testresources</id>
         </profile>
-        <profile>
-            <id>libnd4jhome-defined</id>
```

Comment:
Not sure, that we can remove these two profiles: libnd4jhome-defined, libnd4jhome-undefined.
Libnd4j is not a dependency of nd4j, which means that there could be a chance that developer didn't provide LIBND4J_HOME env variable...